<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="drill">
         <attribute name="title">
          <string>训练</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,12,4">
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QPushButton" name="OpenIma_3">
               <property name="text">
                <string>OpenIma</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="OpenDir_3">
               <property name="text">
                <string>OpenDir</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="OpenVid_3">
               <property name="text">
                <string>OpenVid</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_8">
               <property name="text">
                <string>Next</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_9">
               <property name="text">
                <string>Prew</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QGraphicsView" name="graphicsView_2"/>
             </item>
             <item>
              <widget class="QListView" name="listView_3"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9" stretch="0,3,2">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>文件目录</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_3">
                 <property name="placeholderText">
                  <string>搜索</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListView" name="listView_4"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>权值文件</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_4">
                 <property name="placeholderText">
                  <string>搜索</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListView" name="listView_5"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>进度条</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSlider_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="transparency_2">
                 <property name="text">
                  <string>透明度</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSlider_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_2"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="forecast">
         <attribute name="title">
          <string>预测</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QTabWidget" name="item">
              <property name="tabPosition">
               <enum>QTabWidget::West</enum>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <property name="elideMode">
               <enum>Qt::ElideNone</enum>
              </property>
              <property name="usesScrollButtons">
               <bool>true</bool>
              </property>
              <property name="documentMode">
               <bool>false</bool>
              </property>
              <property name="tabsClosable">
               <bool>false</bool>
              </property>
              <property name="movable">
               <bool>false</bool>
              </property>
              <property name="tabBarAutoHide">
               <bool>false</bool>
              </property>
              <widget class="QWidget" name="image">
               <attribute name="title">
                <string>打开图片</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="1,12,4">
                <item>
                 <widget class="QGroupBox" name="groupBox_11">
                  <property name="title">
                   <string/>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_16">
                   <item>
                    <widget class="QPushButton" name="OpenImage">
                     <property name="text">
                      <string>打开图片</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="OpenDir">
                     <property name="text">
                      <string>打开文件夹</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="predictImage">
                     <property name="text">
                      <string>预测 标记</string>
                     </property>
                     <property name="autoRepeat">
                      <bool>false</bool>
                     </property>
                     <property name="autoDefault">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="Prev_Image">
                     <property name="text">
                      <string>Prev</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="Nest_Image">
                     <property name="text">
                      <string>Nest</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="dir_loadlogs">
                     <property name="text">
                      <string>权值文件</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QDockWidget" name="dockwidget_3">
                  <property name="floating">
                   <bool>false</bool>
                  </property>
                  <property name="features">
                   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
                  </property>
                  <property name="allowedAreas">
                   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea|Qt::TopDockWidgetArea</set>
                  </property>
                  <property name="windowTitle">
                   <string>image</string>
                  </property>
                  <widget class="QWidget" name="dockWidgetContents_4">
                   <layout class="QVBoxLayout" name="verticalLayout_17" stretch="12,12">
                    <item>
                     <widget class="QGraphicsView" name="originImg">
                      <property name="renderHints">
                       <set>QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
                      </property>
                      <property name="dragMode">
                       <enum>QGraphicsView::ScrollHandDrag</enum>
                      </property>
                      <property name="transformationAnchor">
                       <enum>QGraphicsView::AnchorUnderMouse</enum>
                      </property>
                      <property name="resizeAnchor">
                       <enum>QGraphicsView::AnchorUnderMouse</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QGraphicsView" name="predictImg"/>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_12">
                  <property name="title">
                   <string/>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_18">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_19" stretch="0,0,0">
                     <item>
                      <widget class="QLabel" name="label_12">
                       <property name="text">
                        <string>文件目录</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="lineEdit_7">
                       <property name="placeholderText">
                        <string>搜索</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QListWidget" name="pre_list"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_20">
                     <item>
                      <widget class="QLabel" name="label_13">
                       <property name="text">
                        <string>权值文件</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="dir_line">
                       <property name="placeholderText">
                        <string>点击列表，选择权值文件</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QListWidget" name="logsList"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="video">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <attribute name="title">
                <string>打开视频</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,12,4">
                <item>
                 <widget class="QGroupBox" name="groupBox_9">
                  <property name="title">
                   <string/>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_15">
                   <item>
                    <widget class="QPushButton" name="OpenVid">
                     <property name="text">
                      <string>打    开</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="predictVid">
                     <property name="text">
                      <string>预    测</string>
                     </property>
                     <property name="autoRepeat">
                      <bool>false</bool>
                     </property>
                     <property name="autoDefault">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="vid_loadlogs">
                     <property name="text">
                      <string>权值文件</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox">
                  <property name="title">
                   <string/>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <widget class="QVideoWidget" name="ori_video" native="true"/>
                   </item>
                   <item>
                    <widget class="QVideoWidget" name="pre_video" native="true"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_10">
                  <property name="title">
                   <string/>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>权值文件</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="vid_line">
                     <property name="text">
                      <string>点击列表，选择权值文件</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QListWidget" name="vid_list"/>
                   </item>
                   <item>
                    <widget class="QSlider" name="horizontalSlider">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_14">
                     <item>
                      <widget class="QPushButton" name="play_button">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="pause_button">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="stop_button">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1054</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
